### ISSUE TITLE
[Accessibility] Missing the items displayed for CheckedListBox control in the inspect tool 

### ISSUE THREAD

**Issue by Zheng-Li01** [dotnet/winforms#4121]
* More info

1. The issue also can **reproduce on .NET 5.0 RC2.**
2. The issue can't reproduce on .NET SDK 3.1 and .NET 4.8.

* .NET Core Version: 
.NET SDK 6.0.100-alpha.1.20514.15

* Have you experienced this same bug with .NET Framework?: 
No

**Problem description:**
Missing the items displayed for CheckedListBox control in the inspect tool. 

**Minimal repro:**

1. Create a .NET Core project with a CheckedListBox control.
2. Build and run the core project.
3. Observe the checkedListBox1 control in the Inspect tool on UI Automation mode.

**Expected behavior:**
The items displayed correctly for checkedListBox1 control in the Inspect tool.
![CheckedListBoxExpected](https://user-images.githubusercontent.com/38325459/96222886-9cdf9280-0fbf-11eb-8787-a91136fec7be.PNG)

**Actual Behavior:**
Missing the items displayed for checkedListBox1 control in the inspect tool.
![CheckedListBoxActual](https://user-images.githubusercontent.com/38325459/96222894-a1a44680-0fbf-11eb-8911-22f6ebfe3b26.PNG)

**Comment by Tanya-Solyanik** [comment]
@merriemcgaw - this regression makes checked list box unusable under UIA - we should consider servicing.
@Zheng-Li01 - great bug! Do you happen to know when this regressed?

**Comment by merriemcgaw** [comment]
Yes, this is something we need to service.

**Comment by Zheng-Li01** [comment]
@Tanya-Solyanik, verified the issue from 5.0.rc2 to  5.0.100-preview.1.20155.7(https://dotnet.microsoft.com/download/dotnet/5.0), the issue reproduce all the time.

**Comment by merriemcgaw** [comment]
Drat! We will get this in for 6.0 with priority and back port if needed. @vladimir-krestov - this will be on the priority list.

**Comment by RussKie** [comment]
@merriemcgaw do we want to service this in 5.0.1?

**Comment by merriemcgaw** [comment]
Yes, I agree with @Tanya-Solyanik here. We have to service or CheckedListBoxes are entirely unusable under UIA. @Pilchie for FYI.

**Comment by RussKie** [comment]
Raised https://github.com/dotnet/winforms/pull/4209 for servicing.

**Comment by Jenna-Zheng** [comment]
Verified this issue with .NET SDK 6.0.100-alpha.1.20562.2, it is fixed that the items displayed correctly for checkedListBox1 control in the Inspect tool.
![image](https://user-images.githubusercontent.com/64949616/99023799-4ce4f300-25a0-11eb-89a2-455675d11d55.png)

**Comment by Zheng-Li01** [comment]
Verified this issue in .Net SDK 5.0.101 build, this issue is fixed.
